pyproject.toml

```diff
@@ -14,7 +14,7 @@ repository = "https://github.com/STATWORX/statworx-theme"
 homepage = "https://statworx-theme.readthedocs.io/en/latest"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = "~3.11"
```

Comment:
Why do we limit it to Pyhotn 3.11 only?

Reply:
I guess I was moving to quickly here.

With the prior dependency an update of matplotlib to the required version was not possible, ayn ideas why?

Reply:
The version bump action failed anyhow, so deployment isn't done yet